index.js

```diff
@@ -1,12 +1,85 @@
 const express = require('express');
 const helmet = require('helmet');
 
+const knex = require('knex');
+
+const knexConfig = {
+  client: 'sqlite3',
+  useNullAsDefault: true, //needed for sqlite
+  connection: {
+    filename: './data/lambda.sqlite3'
+  }
+}
+const db = knex(knexConfig);
+
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
 
 // endpoints here
+server.get('/api/zoos', async(req, res) => {
+  try {
+    const zoos = await db('zoos');
+    res.status(200).json(zoos)
+  } catch (error) {
+    res.status(500).json(error)
+  }
+})
+
+server.get('/api/zoos/:id', async(req, res) => {
+  try {
+    const zoo = await db('zoos').where({ id: req.params.id }).first('name');
+    res.status(200).json(zoo)
+  } catch (error) {
+  }
```

Comment:
I would like to see a couple of things here.
- Logic that returns a 404 status code and message when the specified id does not exist
- Filled out catch (error) block returning the 500 status code